### DISCUSSION TITLE
My-CS-Notebook/PL/Go/Go-01.%20Go%E8%AF%AD%E8%A8%80%E7%9A%84%E5%9F%BA%E7%A1%80%E8%AF%AD%E6%B3%95/

### DISCUSSION THREAD

**Discussion by giscus** [Zhang-Each/My-CS-Notebook#11] [General]
# My-CS-Notebook/PL/Go/Go-01.%20Go%E8%AF%AD%E8%A8%80%E7%9A%84%E5%9F%BA%E7%A1%80%E8%AF%AD%E6%B3%95/

https://zhang-each.github.io/My-CS-Notebook/PL/Go/Go-01.%20Go%E8%AF%AD%E8%A8%80%E7%9A%84%E5%9F%BA%E7%A1%80%E8%AF%AD%E6%B3%95/

**Comment by Zhang-Each** [comment]
今天翻看自己的博客突然发现当时想干的事情最后烂尾了，最后Go语言相关的东西啥也没学到。
